docs/output.md

```diff
@@ -1,9 +1,5 @@
 # nf-core/pangenome: Output
 
-## :warning: Please read this documentation on the nf-core website: [https://nf-co.re/pangenome/output](https://nf-co.re/pangenome/output)
```

Comment:
Why is this removed? No longer part of the template?

Reply:
Yes, this came in as a part of the Template update.